apps/prairielearn/src/lib/workspace.ts

```diff
@@ -509,20 +563,16 @@ async function initialize(workspace_id: string): Promise<InitializeResult> {
 
   const allWorkspaceFiles = staticFiles.concat(templateFiles).concat(dynamicFiles);
 
-  const root = config.workspaceHomeDirRoot;
-  const destinationPath = path.join(root, remotePath);
-  const sourcePath = `${destinationPath}-${uuidv4()}`;
-
-  await fs.ensureDir(sourcePath);
+  await fs.ensureDir(targetPath);
   await fsPromises.chown(
-    sourcePath,
+    targetPath,
     config.workspaceJobsDirectoryOwnerUid,
     config.workspaceJobsDirectoryOwnerGid,
   );
 
   if (allWorkspaceFiles.length > 0) {
     await async.eachSeries(allWorkspaceFiles, async (workspaceFile) => {
-      const sourceFile = path.join(sourcePath, workspaceFile.name);
+      const sourceFile = path.join(targetPath, workspaceFile.name);
```

Comment:
It seems to me like this is actually `targetFile`? Is that right?

Reply:
Renamed in 9b04c2a07512a4d8049c2373a8598dcc9232d705. This was just a name from the original implementation, where "sourcePath" refers to the temporary path used in the initialization and "destinationPath" refers to the official directory mounted to the workspace. In this case `targetPath` refers to the directory used by the helper function, which happens to be the `sourcePath` in the original initialization function.